packages/types-augment/src/registry.ts

```diff
@@ -514,9 +519,6 @@ declare module '@polkadot/types/types/registry' {
     InstanceDetails: InstanceDetails;
     InstanceId: InstanceId;
     InstanceMetadata: InstanceMetadata;
-    InstantiateRequest: InstantiateRequest;
-    InstantiateReturnValue: InstantiateReturnValue;
-    InstantiateReturnValueTo267: InstantiateReturnValueTo267;
```

Comment:
The issue with removing these is that we break all code of those using overrides. It is fine adding new, but we cannot just drop old aliasses.

Reply:
Would it be possible to just mention it as a breaking change when released?